planning/route_handler/src/route_handler.cpp

```diff
@@ -1054,9 +1060,7 @@ lanelet::ConstLineStrings3d RouteHandler::getFurthestLinestring(
 bool RouteHandler::getLaneChangeTarget(
   const lanelet::ConstLanelets & lanelets, lanelet::ConstLanelet * target_lanelet) const
 {
-  for (auto it = lanelets.begin(); it != lanelets.end(); ++it) {
-    const auto lanelet = *it;
-
+  for (const auto & lanelet : lanelets) {
     int num = getNumLaneToPreferredLane(lanelet);
```

Comment:
Can it be made to constant value?
```suggestion
    const int num = getNumLaneToPreferredLane(lanelet);
```